lib/job-simulation.rb

```diff
@@ -5,10 +5,32 @@ class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    @workers = Stack.new
+    jobs_available.times do |number|
+      @workers.push("Worker ##{number + 1}")
+    end
+    # @workers = jobs_available
+    @waiting = Queue.new
+    until job_seekers == jobs_available
+    @waiting.enqueue("Worker ##{job_seekers}")
+    job_seekers -= 1
+    end
```

Comment:
This is a creative solution, but I think it's a little hard to read. Another possibility is:

```
num_waiting = job_seekers - jobs_available
num_waiting.times do |number|
  @waiting.enqueue("Worker ##{number+1+jobs_available}")
end
```